### PR TITLE
F-71E Test-33899 fixes

### PR DESCRIPTION
**Pull request by kachhwahadivya**
Signed-off-by: Divya Kachhwaha <divya.kachhwaha@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by kachhwahadivya** [comment]
Test is failing even after stateful set
Defect CORTX-33622
FAILED tests/csm/rest/test_capacity_usage.py::TestSystemCapacity::test_33899 - AssertionError: Summation check Passed.healthy byte count check passed.

**Comment by kachhwahadivya** [comment]
Successfully unmounted directory

======================================================= 2468 tests collected in 12.45s =======================================================
[root@ssc-vm-g3-rhev4-1585 cortx-test]#

File: libs/csm/rest/csm_rest_query_deployment.py [hunk 1]
```diff
@@ -411,3 +411,30 @@ def verify_certificate_details(self, expected_response=HTTPStatus.OK):
...
+        return SNS value using Query deployment table
+        """
+        resp = self.get_storage_topology()
+        assert resp.status_code == HTTPStatus.OK, "Status code check failed."
+        data = resp.json()['topology']['storage_sets'][0]['durability']['metadata']
+        data = data.split("+")
+        dvalue = int(data[0])
+        ivalue = int(data[1])
+        xvalue = int(data[2])
+        return dvalue, ivalue, xvalue
```
Comment (RajeshPratapSInghChouhan) [inline libs/csm/rest/csm_rest_query_deployment.py:440]: please add one empty line at eof

File: tests/csm/rest/test_capacity_usage.py [hunk 2]
```diff
@@ -263,14 +233,14 @@ def test_33899(self):
 
         self.log.info("[START] Failure loop")
         for failure_cnt in range(1, self.kvalue + 2):
```
Comment (RajeshPratapSInghChouhan) [inline tests/csm/rest/test_capacity_usage.py:235]: what is the significance of + 2 ? if it is fixed, can we add to const ?

**Review by RajeshPratapSInghChouhan — APPROVED** [review]

File: tests/csm/rest/test_capacity_usage.py (hunk 2 above)
Comment (kachhwahadivya) [reply tests/csm/rest/test_capacity_usage.py:235]: It is fixed to 2 as we want to create one more than K value,

**Review by aditipande96 — APPROVED** [review]

**Comment by kachhwahadivya** [comment]
[collection.log](https://github.com/Seagate/cortx-test/files/9555431/collection.log)

**Review by sonalk0209 — APPROVED** [review]
Approving for merge!!!!

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**